app/src/common/utils/validation/validate.js

```
@@ -30,6 +30,8 @@ import { MAX_FIELD_LENGTH } from './constants';
 
 const URL_REGEX = /^(ftp|http|https):\/\/[^\s"]+[^\s".]$/;
 
+const URL_REGEX = /^(ftp|http|https):\/\/[^\s"]+[^\s".]$/;
```

Biome (2.3.13)

[error] Shouldn't redeclare 'URL_REGEX'. Consider to delete it or rename it.

'URL_REGEX' is defined here:


(lint/suspicious/noRedeclare)